### PR TITLE
295 Added favicon on heartbeat pages

### PR DESCRIPTION
**Pull request by cedricabuso**

### CODE REVIEW

File: room.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
 
     <head>
+        <link rel="icon" type="image/png" href="../images/favicon-hb-16x16.png">
```
Comment (nhvu1988) [inline room.html]: I think it should be 
```html
<link rel="icon" type="image/png" href="images/favicon-hb-16x16.png">
```

File: auth/index.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <html>
 
 <head>
+    <link rel="icon" type="image/png" href="../images/favicon-hb-16x16.png">
```
Comment (nhvu1988) [inline auth/index.html]: please add title for this page?

File: music-play-stats/index.html [hunk 3]
```diff
@@ -1,9 +1,10 @@
 <html>
-<header>
+<head>
+    <link rel="icon" type="image/png" href="../images/favicon-hb-16x16.png">
```
Comment (nhvu1988) [inline music-play-stats/index.html]: please add title for this page?

**Comment by cedricabuso** [comment]
@nhvu1988 Updated :)

**Pull request merged**